src/gl/opengl/vcFramebuffer.cpp

```diff
@@ -54,28 +54,58 @@ void vcFramebuffer_Destroy(vcFramebuffer **ppFramebuffer)
   VERIFY_GL();
 }
 
-bool vcFramebuffer_Bind(vcFramebuffer *pFramebuffer)
+bool vcFramebuffer_Bind(vcFramebuffer *pFramebuffer, const vcFramebufferClearOperation clearOperation /*= vcFramebufferClearOperation_None*/, uint32_t clearColour /*= 0x0*/, const vcFramebufferClearOperation clearPreviousOperation /*= vcFramebufferClearOperation_None*/)
 {
-  if (pFramebuffer == nullptr || pFramebuffer->id == GL_INVALID_INDEX)
+  if (pFramebuffer == nullptr)
+    return false;
+
+  // OpenGL ES only
```

Comment:
This creates a discrepancy between the GLs, what does that mean? Also, what about WebGL?

Reply:
Valid points.

We should have a discussion about how best to handle this sort of targeted functionality.